src/ucs/debug/log.c

```diff
@@ -314,6 +314,30 @@ const char * ucs_log_dump_hex(const void* data, size_t length, char *buf,
     return buf;
 }
 
+const char * ucs_log_dump_hex_mline(const void* data, size_t length,
```

Comment:
can you please move this function, along with ucs_log_dump_hex, to string.c in ucs?
because these functions are generic string and not necessarily related to logging
we can export only `ucs_string_dump_hex(const void* data, size_t length, char *buf, size_t max, size_t per_line)`
if per_line is SIZE_MAX it will behave as single line dump

Reply:
ok, moved to string.h/.c, updated arg list